config/development.json

```diff
@@ -148,8 +148,18 @@
 		"perfmon": false,
 		"mailing-lists/unsubscribe": true,
 
-		"jetpack/calypso-first-signup-flow": true
+		"jetpack/calypso-first-signup-flow": true,
+		"sync-handler": true
 	},
+
+	"sync-handler-defaults": {
```

Comment:
Agree but at the moment `sync-handler` is only enabled in development environment. I think we should wait once it is enabled (I hope) in other environments.